Code/Framework/AzCore/AzCore/Memory/AllocatorManager.h

```diff
@@ -167,8 +167,17 @@ namespace AZ
             }
         };
 
+        struct AllocationMetadata
+        {
+            const AZ::AllocatorManager::CodePoint* m_codePoint;
+            uint64_t m_mask;
+            unsigned int m_tag;
+            const char* m_assetName;
+            size_t m_assetSizeLimit;
+        };
```

Comment:
This is what we have instead of tuple. I use asset size limit to avoid taking large (like 64k) page allocation via page allocators to particular asset accounts. That memory is untracked.